scripts/docker-run.sh

```diff
@@ -1,3 +1,9 @@
 #!/usr/bin/env bash
```

Comment:
I also think it's better to remove these 2 files from the PR, so that we can keep this PR clean.

If we want to add build tools for specital platforms, we can do it in another PR and document it in README.md.